### ISSUE TITLE
Bracken requires a non-standard NCBI taxonomy format

### ISSUE THREAD

**Issue by davve2** [jenniferlu717/Bracken#127]
I´m developing a program that makes use of kraken2/bracken downstream, however I discovered that kraken2 add it´s own names.dmp and nodes.dmp format by adding a column with a dash "-" into the files. This causes problems when using softwares written to use standard NCBI format. 

This dependency follows though to Bracken that later on requires this dash when parsing a taxonomy. 

If the dash is not present it creates the following issues #121 and #126.

**Comment by larssnip** [comment]
I have made many kraken2/bracken databases, where I have edited/changed the names.dmp and nodes.dmp, and I have never had a column with a dash inside them? Without any problems. I follow the NCBI format strictly, with this format for names.dmp
|\ttax_id\t|\tname_txt\t|\tunique_name\t|\tname_class\t|

For nodes.dmp it is 13 columns, separated by \t|\t but I believe only the 3 first are used...? Exactly where is this column with a dash?

**Comment by davve2** [comment]
Thank you for this clarification @larssnip 

This issue is originally from a user of my program, where I try to make a unified taxonomy (from all the different formats), one of my users claimed that they got an error with Bracken and could only solve it by using dashes in the format.

I did unfortunately not investigate this issue myself (and still haven´t gotten to that point), so perhaps it was a user error, rather than a real error. I will be more careful in the future to lift problems I did not thoroughly investigate myself!
